rest/api.go

```diff
@@ -231,6 +231,17 @@ func (r *muxAPI) preprocess() {
 	}
 }
 
+// Check the route for an error and log the error if it exists.
+func (r *muxAPI) checkRoute(method, uri string, route *mux.Route) {
+	err := route.GetError()
+
+	if err != nil {
+		log.Printf("Failed to setup route %s with %v", uri, err)
+	} else {
+		r.config.Debugf("Registered create handler at %s %s", method, uri)
```

Comment:
Probably want to log out the correct type of handler. Otherwise this logs out "create handler" for every route.